lib/lamby/debug.rb

```diff
@@ -6,7 +6,7 @@ module Debug
 
     def on?(event)
       params = event['multiValueQueryStringParameters'] || event['queryStringParameters']
-      (Rails.env.development? || ENV['LAMBY_DEBUG']) && params && params['debug'] == '1'
+      (development? && ENV['LAMBY_DEBUG']) && params && params['debug'] == '1'
```

Comment:
@metaskills - I realized I change this condition fro an `or` to an `and`.

I can change it back - I'm assuming you want development mode to be enough to trigger Debug mode?

Reply:
I changed this logic back to what it originally was!